include/zephyr/zbus/zbus.h

```diff
@@ -263,6 +267,30 @@ struct zbus_channel_observation {
 #define _ZBUS_RUNTIME_OBSERVERS_DECL(_name)
 #endif /* CONFIG_ZBUS_RUNTIME_OBSERVERS */
 
+#define _ZBUS_MESSAGE_NAME(_name) _CONCAT(_zbus_message_, _name)
+
+#define _ZBUS_CHAN_DEFINE(_name, _id, _type, _validator, _user_data)                               \
```

Comment:
Can you put the `clang-format off and on` to avoid indentation issues? The GitHub action identifies that.